.github/workflows/triage.yml

```
@@ -20,42 +19,105 @@ jobs:
+                        ... on ProjectV2Field {
+                          id
+                          name
+                        }
+                        ... on ProjectV2SingleSelectField {
+                          id
+                          name
+                          options {
+                            id
+                            name
+                          }
+                        }
                       }
                     }
                   }
                 }
               }' > project_columns.json
-          echo 'COLUMN_ID='$(jq '.data.repository.project.columns.nodes[] | select(.name == "Triage") | .id' project_columns.json) >> $GITHUB_ENV
-          echo 'COLUMN_NAME='$(jq '.data.repository.project.columns.nodes[] | select(.name == "Triage") | .name' project_columns.json) >> $GITHUB_ENV
+          echo 'PROJECT_ID='$(
+            jq \
+              '.data.organization.projectV2.id' \
+              project_columns.json
+          ) >> $GITHUB_ENV
+          echo 'STATUS_FIELD_ID='$(
+            jq \
+              '.data.organization.projectV2.fields.nodes[]
+              | select(.name== "Status")
+              | .id' \
+              project_columns.json
+          ) >> $GITHUB_ENV
+          echo 'TRIAGE_COLUMN_ID='$(
+            jq --raw-output \
+              '.data.organization.projectV2.fields.nodes[]
+              | select(.name== "Status")
+              | .options[]
+              | select(.name=="To Triage")
+              | .id' \
+              project_columns.json
+          ) >> $GITHUB_ENV
 
-      - name: Add issue#${{ github.event.issue.number }} to Triage column
+      - name: Add issue#${{ github.event.issue.number }} to project
         env:
           ISSUE_ID: ${{ github.event.issue.node_id }}
         run: |
-          echo "Adding issue: ${{ github.event.issue.number }} to column $COLUMN_NAME in project $PROJECT_NUMBER"
-          if [[ $DRY_RUN == "false" ]]; then
+          echo "Adding issue: ${{ github.event.issue.number }} to project $PROJECT_NUMBER"
+          if [[ $DRY_RUN == 'false' ]]; then
+            item_id="$(
+              gh api graphql \
+                --jq '.data.addProjectV2ItemById.item.id' \
+                --raw-field project_id=$PROJECT_ID \
+                --raw-field issue_id=$ISSUE_ID \
+                --raw-field query='
+                mutation($project_id: ID!, $issue_id: ID!) {
+                  addProjectV2ItemById(input: {projectId: $project_id, contentId: $issue_id}) {
+                    item {
+                      id
+                    }
+                  }
+                }'
+            )"
+            echo "ITEM_ID=$item_id" >> $GITHUB_ENV
+          fi
+
+      - name: Move issue#${{ github.event.issue.number }} to Triage column
+        run: |
+          echo "Moving issue: ${{ github.event.issue.number }} to Triage column in project $PROJECT_NUMBER"
+          if [[ $DRY_RUN == 'false' ]]; then
```

Also here, not sure, but it seems it should be:
```suggestion
          if [[ $DRY_RUN == 'true' ]]; then
```

---

See: https://github.com/ethereum/solidity/pull/13962#discussion_r1107553809